azure/eventhub_log_forwarder/activity_log_diagnostic_settings.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "settingName": {
+            "type": "string",
+            "metadata": {
```

Comment:
Let's set a default for this - what about `datadog-activity-logs-diagnostic-setting` or something like that